src/build_and_deploy.sh

```diff
@@ -0,0 +1,76 @@
+#
+# The `build_and_deploy.sh` is the entrypoint for `kw bd`, i.e.,
+# `kw build` followed by `kw deploy`.
+#
+
+include "${KW_LIB_DIR}/deploy.sh"
+include "${KW_LIB_DIR}/build.sh"
+include "${KW_LIB_DIR}/lib/kwio.sh"
+
+# `kw bd` is a shortcut for `kw build` followed `kw deploy`, to run the entire workflow
+# of building a kernel from source and deploying it to a target machine.
+#
+# @raw_options String with all deploy options
+function build_and_deploy_main()
+{
+  if [[ "$1" =~ -h|--help ]]; then
+    build_and_deploy_help "$1"
+    exit 0
+  fi
+
+  parse_build_and_deploy_options "$@"
+  if [[ "$?" -gt 0 ]]; then
+    complain "${options_values['ERROR']}"
+    build_and_deploy_help
+    return 22 # EINVAL
+  fi
+
+  build_kernel_main
+  if [[ "$?" != 0 ]]; then
+    complain '%s\n' 'kernel build failed'
+    exit 22 # EINVAL
+  fi
+  deploy_main 1 "$@"
```

Comment:
Add a new line after the `fi` part. Also, add a comment that explains what  `1` parameter means.